modules/odo-getting-started-delete.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-nodejs.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-dot-net.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-go.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-java.adoc
+
+:_content-type: Procedure
+[id="odo-getting-started-delete_{context}"]
+
+= Deleting the resources (optional)
+
+After testing your application, you may optionally remove the resources by using the `odo delete component` command.
+
+
+.Procedure
+* Run the `odo delete component` command:
++
+.Sample command
+[source, terminal]
+----
+$ odo delete component
```

Comment:
```suggestion
$ odo delete <component>
```
Use angle brackets to indicate user-specified values.

Reply:
No. The component is inferred from the current directory, so this is not user input - it is the actual command you use.

```
$ odo delete component --help
Delete component

Usage:
  odo delete component [flags]

Examples:
  # Delete the component present in the current directory from the cluster
  odo delete component
  
  # Delete the component named 'frontend' in the currently active namespace from the cluster
  odo delete component --name frontend
  
  # Delete the component named 'frontend' in the 'myproject' namespace from the cluster
  odo delete component --name frontend --namespace myproject

Flags:
  -f, --force              Delete component without prompting
  -h, --help               Help for component
      --name string        Name of the component to delete, optional. By default, the component described in the local devfile is deleted
      --namespace string   Namespace in which to find the component to delete, optional. By default, the current namespace defined in kubeconfig is used
  -w, --wait               Wait for deletion of all dependent resources

Additional Flags:
      --kubeconfig string    Paths to a kubeconfig. Only required if out-of-cluster.
  -o, --o string             Specify output format, supported format: json
  -v, --v Level              Number for the log level verbosity. Level varies from 0 to 9 (default 0).
      --vmodule moduleSpec   Comma-separated list of pattern=N settings for file-filtered logging

```